scenarios/post_action_prometheus.yml

```diff
@@ -0,0 +1,22 @@
+config:
+  runStrategy:
+    runs: 1
+    maxSecondsBetweenRuns: 10
+    minSecondsBetweenRuns: 1
+scenarios:
+  - name: "check 2 pods are in namespace with selector: prometheus"
+    steps:
+    - podAction:
+        matches:
+          - labels:
+              namespace: "openshift-monitoring"
+              selector: "app=prometheus"
+        filters:
+          - property:
+              name: "state"
+              value: "Running"
+        # The actions will be executed in the order specified
+        actions:
+          - checkPodCount:
```

Comment:
Is there a way to mention a timeout for the check pod count action? cc: @paigerube14

Reply:
Not that I can see, I will keep looking though

Reply:
We might want to open an issue in powerfulseal to add support to specify the retries as this will avoid maintaining two configs for the same scenario. Kill can be the first action and checkPodCount can be the next action with retries in place. Thoughts?